src/pages/CommitDetailPage/CommitDetailPage.jsx

```diff
@@ -66,6 +70,9 @@ function CommitDetailPage() {
     return <NotFound />
   }
 
+  const hideCommitSummary =
+    repoSettings?.repository?.private && tierName === TierNames.TEAM
```

Comment:
I would say @nicholas-codecov, I'd probably leave this req out for now as I'm not 100% sure if we're going to do this for the V0 of the project. I think it's something we want to, but it wasn't discussed initially so I want to hear from Product and whatnot to see how crucial it is V0